File: system-test/dotnet/SystemTests/Legacy/Mod08/SmokeTests/System/MyShopSmokeTest.cs
using SystemTests.Legacy.Mod08.Base;
using Optivem.Testing;
using Xunit;

namespace SystemTests.Legacy.Mod08.SmokeTests.System;

public class MyShopSmokeTest : BaseScenarioDslTest
{
    [Theory]
    [ChannelData(ChannelType.UI, ChannelType.API)]
    public async Task ShouldBeAbleToGoToMyShop(Channel channel)
    {
        await Scenario(channel)
            .Assume().MyShop().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod07/smoke/system/MyShopSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod07.smoke.system;

import com.mycompany.myshop.systemtest.legacy.mod07.base.BaseUseCaseDslTest;
import com.mycompany.myshop.testkit.channel.ChannelType;
import com.optivem.testing.Channel;
import org.junit.jupiter.api.TestTemplate;

class MyShopSmokeTest extends BaseUseCaseDslTest {
    @TestTemplate
    @Channel({ChannelType.UI, ChannelType.API})
    void shouldBeAbleToGoToMyShop() {
        app.myShop().goToMyShop()
                .execute()
                .shouldSucceed();
    }
}

